Basic-Car-Maintenance/Shared/Dashboard/Views/ExportOptionsView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ExportOptionsView.swift
+//  Basic-Car-Maintenance
+//
+//  https://github.com/mikaelacaron/Basic-Car-Maintenance
+//  See LICENSE for license information.
+//
+
+import SwiftUI
+import PDFKit
+
+struct ExportOptionsView: View {
+    @Environment(\.dismiss) var dismiss
+    @State private var selectedVehicle: Vehicle?
+    @State private var isShowingShareSheet = false
```

Comment:
renamed to `isShowingThumbnail` because it's not technically showing the share sheet yet, the `ShareLink` does that